backend/services/interfaces/testSessionService.ts

```diff
@@ -34,18 +29,18 @@ export interface TestSessionResponseDTO {
   /** the unique identifier for the test session */
   id: string;
   /** the corresponding test from the Test collection */
-  test: TestResponseDTO;
+  test: string;
   /** the teacher administering the test from the User collection */
```

Comment:
nit: (& same for entries below)
```suggestion
  /** the id of the teacher administering the test from the User collection */
```

Reply:
Copilot not yet smart enough to generate fully correct comments, I see 🤓